src/components/my-app.js

```diff
@@ -239,20 +240,20 @@ class MyApp extends connect(store)(LitElement) {
     setPassiveTouchGestures(true);
```

Comment:
As we import:

`import { setPassiveTouchGestures } from '@polymer/polymer/lib/utils/settings.js';`

We should add `@polymer/polymer` to the `package.json`, right?